tests/datasets/test_eudem.py

```diff
@@ -33,21 +33,23 @@ def test_getitem(self, dataset: EUDEM) -> None:
         assert isinstance(x["mask"], torch.Tensor)
 
     def test_extracted_already(self, dataset: EUDEM) -> None:
-        zipfile = os.path.join(dataset.root, "eu_dem_v11_E30N10.zip")
-        shutil.unpack_archive(zipfile, dataset.root, "zip")
-        EUDEM(dataset.root)
+        zipfile = dataset.list_files(filename_glob="eu_dem_v11_E30N10.zip")[0]
+        outdir = os.path.abspath(os.path.join(zipfile, os.pardir))
+        # TODO: This changes the behaviour from unpacking in root to same dir as archive
+        shutil.unpack_archive(zipfile, outdir, "zip")
```

Comment:
Why is this hack necessary? Can't we do:
```suggestion
        zipfile = os.path.join(dataset.paths[0], "eu_dem_v11_E30N10.zip")
        shutil.unpack_archive(zipfile, dataset.paths[0], "zip")
        EUDEM(dataset.root)
```

Reply:
Yes, that works if the zip archive is (forever) in `dataset.paths[0]` and not in subdirectory.

Reply:
It will forever be in this location, at least for our unit tests.